sync/src/block/extension.rs

```diff
@@ -173,6 +173,40 @@ impl Extension {
         }
     }
 
+    fn send_status(&mut self, id: &NodeId) {
+        let chain_info = self.client.chain_info();
+        self.api.send(
+            id,
+            Arc::new(
+                Message::Status {
+                    nonce: U256::from(self.nonce),
+                    best_hash: chain_info.best_proposal_block_hash,
+                    genesis_hash: chain_info.genesis_hash,
+                }
+                .rlp_bytes(),
+            ),
+        );
+        self.nonce += 1;
+    }
+
+    fn send_status_broadcast(&mut self) {
```

Comment:
Let's rename this bo `broadcast_status`. I think `send` and `broadcast` are redundant.

Reply:
I wanted to match the naming convention around the function. They all start with `send_`

Reply:
Hmm. I'm not convinced enough, but you have a point too. Let's leave it as is for now then.